addon/model/model-range.ts

```diff
@@ -30,6 +30,11 @@ export default class ModelRange {
     const end = ModelPosition.fromInElement(element, endOffset);
     return new ModelRange(start, end);
   }
+  static fromAroundNode(node: ModelNode) {
```

Comment:
nitpicking: could benefit from a newline above the method

Reply:
I love nitpicking, good catch :)